src/diffusers/image_processor.py

```diff
@@ -168,6 +168,20 @@ def postprocess(
         image,
```

Comment:
```suggestion
        image: torch.FloatTensor,
```

The input type to `postprocess` should **always** be a PyTorch tensor I think no?

Reply:
Let's also add a raise Error check that if this is not the case we throw an error